clap-utils/Cargo.toml

```
@@ -27,5 +27,8 @@ tempfile = "3.2.0"
 [lib]
 name = "solana_clap_utils"
 
+[features]
+default = ["solana-remote-wallet"]
```

```suggestion
default = ["remote-wallet"]
```
"solana" is implicit